api/v2.0/swagger.yaml

```diff
@@ -1098,3 +1251,87 @@ definitions:
       op_time:
         type: string
         description: The time when this operation is triggered.
+  Provider:
+    type: object
+    properties:
+      id:
+        type: string
+        description: id
+      name:
+        type: string
+        description: name
+      icon:
+        type: string
+        description: icon
+      maintainers:
+        type: array
+        description: maintainers
+        items:
+          type: string
+      version:
+        type: string
+        description: version
+      source:
+        type: string
+        description: source
+      auth_mode:
+        type: string
+        description: auth mode
+  Instance:
+    type: object
+    properties:
+      id:
+        type: integer
+        description: Unique ID
+      name:
+        type: string
+        description: Instance name
+      description:
+        type: string
+        description: Description of instance
+      provider:
+        type: string
+        description: Based on which driver, identified by ID
+      endpoint:
+        type: string
+        description: The service endpoint of this instance
+      auth_mode:
+        type: string
+        description: The authentication way supported
+      auth_data:
+        type: object
+        description: The auth credential data if exists
+        additionalProperties:
+          type: string
+      status:
+        type: string
+        description: The health status
+      enabled:
```

Comment:
have a new properties: default(represents whether the instance is default)

Reply:
Added.